test/checks/aria/has-widget-role.js

```diff
@@ -0,0 +1,460 @@
+describe('has-widget-role', function () {
```

Comment:
These roles are marked up as widgets in lib/commons/aria/index.js, but not in the aria spec

The ARIA spec seems to differ with axe on these:

## tooltip
https://github.com/dequelabs/axe-core/blame/develop/lib/commons/aria/index.js#L874
https://www.w3.org/TR/wai-aria-1.1/#tooltip

# Window roles
## dialog
https://github.com/dequelabs/axe-core/blame/develop/lib/commons/aria/index.js#L319
https://www.w3.org/TR/wai-aria-1.1/#dialog

## alertdialog
https://github.com/dequelabs/axe-core/blame/develop/lib/commons/aria/index.js#L191
https://www.w3.org/TR/wai-aria-1.1/#alertdialog

# Live region roles
## timer
https://github.com/dequelabs/axe-core/blame/develop/lib/commons/aria/index.js#L855
https://www.w3.org/TR/wai-aria-1.1/#timer

## status
https://github.com/dequelabs/axe-core/blame/develop/lib/commons/aria/index.js#L764
https://www.w3.org/TR/wai-aria-1.1/#status

## marquee
https://github.com/dequelabs/axe-core/blame/develop/lib/commons/aria/index.js#L494
https://www.w3.org/TR/wai-aria-1.1/#marquee

## log
https://github.com/dequelabs/axe-core/blame/develop/lib/commons/aria/index.js#L475
https://www.w3.org/TR/wai-aria-1.1/#log

## alert
https://github.com/dequelabs/axe-core/blame/develop/lib/commons/aria/index.js#L182
https://www.w3.org/TR/wai-aria-1.1/#alert

Reply:
Can you create a new ticket for this?